src/main/java/io/resourcepool/ssdp/client/SsdpParams.java

```diff
@@ -11,7 +11,12 @@
  */
 public class SsdpParams {
   private static InetAddress ssdpMulticastIpv4Address;
-  private static final int SSDP_MULTICAST_PORT = 1900;
+  private static   int SSDP_MULTICAST_PORT = 1900;
```

Comment:
We shouldn't modify static final attributes. Those are constants not made to be changed.
==> keep it as SSDP_MULTICAST_DEFAULT_PORT
==> creating one new attribute in the SsdpOptions called overrideBindingPort which defaults to SSDP_MULTICAST_DEFAULT_PORT would be better and would do the trick. 
In your use case, you only need to override the binding port and not the multicast port from my understanding.